datalakehouse-api/lakehouse_query_api.py

```
@@ -0,0 +1,931 @@
+    gold_df = silver_to_gold_alerts_scalar_only()
+    gold_count = gold_df.count()
+    logger.info(f"  Gold rows: {gold_count}")
+
+    return bronze_count, silver_count, gold_count
+
+
+# ============================================================
+# Helper: Query Hudi data (Gold registry)
+# ============================================================
+
+def _get_hudi_data_sync(table_name: str, filters: dict = None, columns: list = None, limit: int = None):
```

Ruff (0.15.10)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)